CHANGELOG.rst

```diff
@@ -13,6 +13,10 @@ Added:
 
 - Ability to interpolate stimulator output waveforms from start timepoints of each subprotocol
 
+Changed:
+^^^^^^
+- Update use of ``pd.DataFrame.append`` to ``pd.DataFrame.concat``
```

Comment:
I don't think we need a changelog entry for this since nothing is really changing, we're just refactoring how something is done